src/components/Start.js

```diff
@@ -0,0 +1,38 @@
+import React,{ useContext } from 'react';
+import { NavLink } from 'react-router-dom';
+import decoration from '../assets/Decoration.svg';
+import Header from './Header';
+import { LoginContext } from './LoginContext'
+import { FormContext } from './FormContext'
+
+const Start=()=>{
+
+    const { isLoggedIn } = useContext(LoginContext);
+    const { setPage } = useContext(FormContext);
+    
+    return <>
+        <header className='container' id='start'>
+            <div id='header-div-left'>
```

Comment:
What's the point of this? If you need to insert image you can probably use before on next element